src/components/Header/Header.tsx

```diff
@@ -79,12 +86,12 @@ export const Header = () => {
                         target={item.external ? "_blank" : undefined}
                         rel="noopener noreferrer"
                         className={classNames(
-                          item.current
+                          item.current && router.pathname.includes(item.href)
```

Comment:
se vuoi fare così andrebbe aggiunto lo stesso check anche nella versione mobile (ad ora il bottone rimane sempre attivo in qualunque pagina)
![image](https://user-images.githubusercontent.com/15048443/224247717-2674722d-282f-4e89-8c7f-a1cdb6c12391.png)

Reply:
Non avevo notato fossero duplicati, fatto!